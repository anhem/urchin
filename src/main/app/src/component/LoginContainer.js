import React, {Component} from 'react'

class LoginContainer extends Component {
    render() {
        return (
            <h1>
                implement login
            </h1>
        );
    }
}

export default LoginContainer;