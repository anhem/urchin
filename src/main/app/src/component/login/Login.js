import React, {Component} from 'react'

class Login extends Component {

    render() {
        return (
            <div data-view="login"
                 className="login">
                <form method="post" action="/login">
                    <input type="text" name="username" placeholder="Username"/>
                    <input type="password" name="password" placeholder="Password"/>
                    <input type="submit" name="login" value="Login"/>
                </form>
            </div>
        )
    }
}

export default Login