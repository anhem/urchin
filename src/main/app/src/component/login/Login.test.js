import React from 'react';
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json';
import LoginContainer from "./LoginContainer";
import Login from "./Login";

describe("LoginContainer", () => {

    it('match snapshot', () => {
        expect(toJson(shallow(<Login/>))).toMatchSnapshot();
    });
});
