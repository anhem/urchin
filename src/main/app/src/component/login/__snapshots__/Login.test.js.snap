// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer match snapshot 1`] = `
<div
  className="login"
  data-view="login"
>
  <form
    action="/login"
    method="post"
  >
    <input
      name="username"
      placeholder="Username"
      type="text"
    />
    <input
      name="password"
      placeholder="Password"
      type="password"
    />
    <input
      name="login"
      type="submit"
      value="Login"
    />
  </form>
</div>
`;
